packages/image_picker/image_picker/ios/Classes/FLTImagePickerPlugin.m

```diff
@@ -70,6 +69,21 @@ - (UIViewController *)viewControllerWithWindow:(UIWindow *)window {
   return topController;
 }
 
+/**
+ * Applies UIImagePickerControllerCameraDevice on the UIImagePickerController.
+ *
+ * If the cameraDevice value that is fetched from @c arguments is 1 then returns
+ * UIImagePickerControllerCameraDeviceFront. If the cameraDevice value that is fetched
+ * from @c arguments is 0 then returns UIImagePickerControllerCameraDeviceRear.
+ *
+ * @param @arguments that should be used to get cameraDevice value.
```

Comment:
`arguments` is the name of the parameter, not a Doxygen command, so should not have a `@`.

Reply:
This hasn't been changed.

Reply:
Oops, sorry. I also changed other comments with the same mistake. :)